client/clientset/versioned/typed/volumegroupsnapshot/v1beta1/doc.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2024 The Kubernetes Authors.
+Copyright 2025 The Kubernetes Authors.
```

Comment:
Why is this changed?  Same comment in other places.

Reply:
I guess this happened because I regenerated these files.
We could avoid putting this changes into the patch if you think it makes easier to review it.

Reply:
Right.  VolumeGroupSnapshot v1beta1 and VolumeSnapshot v1 related files should not be changed.